core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/PurchasesConfiguration.kt

```diff
@@ -25,13 +25,39 @@ public class PurchasesConfiguration private constructor(
                 "appUserId=$appUserId, " +
                 "purchasesAreCompletedBy=$purchasesAreCompletedBy, " +
                 "userDefaultsSuiteName=$userDefaultsSuiteName, " +
+                "storeKitVersion=$storeKitVersion, " +
```

Comment:
Should we also add the `pendingTransactionsForPrepaidPlansEnabled` option here? We can also do it in a minor later, but it would be great to do it in the same version we bump the native.

Reply:
I think we should add those new BC7 APIs before the v1 stable if possible. That way we can immediately advertise "full BC7 support". Doesn't need to be in this PR though, but can be.

Reply:
Nice! What do you think about adding `installmentsInfo` (see Toni's parent comment)?